### PR TITLE
PDA Resprite

### PR DESCRIPTION
**Pull request by AyIong**
## What Does This PR Do
PDA resprites by @Fridh3lm
Also, detective, ntr and magistrate got their PDA sprites
Moved QM PDA to heads subtype

## Why It's Good For The Game
New fancy PDAs

## Images of changes
image

UI
image

<details><summary> GIF </summary>

dreamseeker_09-10-24_21'16'14

</details>

## Testing
Watched on SDMM and local server, puted ID inside, turn on flashlight and call notify overlay

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.

<hr>

## Changelog

:cl: fridhelm1
imageadd: PDA's got resprited. Also, Magistrate, Detective and NTR got own PDA style
/:cl:

### CODE REVIEW

**Review by PopeDaveThe3th — APPROVED** [review]

**Comment by Christasmurf** [comment]
There are a series of PDA sprites in HUD elements that still look like the old PDAs.
AI Internal PDA system HUD icons
image
image
PDA inventory slot (multiple themes)

All HUDs with PDA icons are located in icons/mob folder, 'screen_ai.dmi' as well as 'screen_midnight.dmi' are the two features in screenshots, however there is also
screen_operative.dmi
screen_plasmafire.dmi
screen_retro.dmi
screen_slimecore.dmi
screen_white.dmi

**Comment by AyIong** [comment]
> Should be RELATIVELY simple as they're all one solid colour so all you really have to do is one of them and then recolour it

I passed this on to the spriter, so it will most likely be done and uploaded in a few days.

File: code/modules/pda/PDA.dm [hunk 1]
```diff
@@ -160,8 +160,12 @@ GLOBAL_LIST_EMPTY(PDAs)
 	. = ..()
 	if(id)
 		. += image('icons/goonstation/objects/pda_overlay.dmi', id.icon_state)
+
 	if(length(notifying_programs))
-		. += image('icons/obj/pda.dmi', "pda-r")
+		if(icon_state == "pda-library")
```
Comment (Henri215) [inline code/modules/pda/PDA.dm:165]: Is it not better to use a method other than icon_state? This can become an oversight in the future.

**Review by Henri215 — COMMENTED** [review]
Do you think the icons/goonstation/objects/pda_overlay.dmi is still considered a goonstation icons file? It all got resprited right?

Everything else is looking good so far.

File: code/modules/pda/PDA.dm (hunk 1 above)
Comment (AyIong) [reply code/modules/pda/PDA.dm:165]: I just don't see a way to make it better

**Comment by AyIong** [comment]
> Do you think the icons/goonstation/objects/pda_overlay.dmi is still considered a goonstation icons file? It all got resprited right?

I haven't looked at them closely, but I think they're just reflected vertically, without any serious resprite

**Review by Henri215 — APPROVED** [review]

**Comment by Scribble-Sheep** [comment]
Rip unergonomic book PDA 😔

**Review by S34NW — APPROVED** [review]

**Pull request merged**